File: code/lib/models/pet.dart
import 'dart:ffi';

class Pet {
  String nome;
  String descricao;
  String image;
  bool isVacinado;
  String idade;

  Pet({
    this.nome = '',
    this.descricao = '',
    this.image = '',
    this.isVacinado = true,
    this.idade = '',
  });

  factory Pet.fromJson(Map<String, dynamic> parsedJson){

    return Pet(
      nome: parsedJson['nome'],
      descricao : parsedJson['descricao'],
      image : parsedJson['image'],
      isVacinado : parsedJson['isVacinado'],
      idade : parsedJson['idade'],
    );
  }
}

File: code/backend/Pet/Pet.js
const Sequelize = require("sequelize");
const connection = require("../database/database");

const Pet = connection.define("pet", {
  nome: {
    type: Sequelize.STRING,
    allowNull: false,
  },
  idade: {
      type: Sequelize.STRING,
      allowNull: false,
  },
  isVacinado: {
      type: Sequelize.BOOLEAN,
      allowNull: false,
  },
  descricao: {
      type: Sequelize.BOOLEAN,
      allowNull: false,
  },
  tipo: {
      type: Sequelize.BOOLEAN,
      allowNull: false,
  },
});

Pet.sync({ force: false });

module.exports = Pet;
